config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -165,23 +133,23 @@ periodics:
           memory: "8Gi"
 
 # manual-release-bump-required
-- interval: 1h
+- interval: 30m
   name: ci-kubernetes-build-stable2
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+    - image: gcr.io/k8s-testimages/bootstrap:v20190213-2f4f06285
       args:
-      - --repo=k8s.io/kubernetes=release-1.11
+      - --repo=k8s.io/kubernetes=release-1.12
       - --repo=k8s.io/release
       - --root=/go/src
       - --timeout=180
       - --scenario=kubernetes_build
       - --
       - --allow-dup
-      - --extra-publish-file=k8s-stable2
+      - --extra-publish-file=k8s-stable1
```

Comment:
job name references `stable2` but this is referencing `stable1`, I'm also unclear on why the interval bump

Reply:
done